src/main/java/io/wispforest/owo/command/debug/DamageCommand.java

```diff
@@ -19,26 +20,24 @@ public class DamageCommand {
 
     public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
         dispatcher.register(literal("damage")
+                .executes(context -> execute(context, DamageSource.OUT_OF_WORLD, context.getSource().getPlayerOrThrow(), 1))
```

Comment:
I believe I said pretty clearly that just damage without any arguments does not make sense